ckanext/unaids/tests/test_dataset_transfer.py

```diff
@@ -97,31 +97,49 @@ def test_get_org_admins_with_no_users_in_org(self, app):
             )
 
     @mock.patch('ckan.lib.mailer.mail_user')
-    def test_send_dataset_transfer_emails(self, mocked_mail_user, app):
-        user_1, user_2 = [
-            factories.User()
-            for x in range(2)
-        ]
-        org_1, org_2 = [
-            factories.Organization(user=user)
-            for user in [user_1, user_2]
-        ]
+    @mock.patch('ckan.lib.mailer._mail_recipient')
```

Comment:
With  a quick check it looks like the mail_receipient patch is not used.
You shouldn't be mocking protected (marked with `_` prefix) functions/fields in general.
After removing this mock test passed just fine. Please have a look.